.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Feature request
+about: Suggest an idea for a new feature
+title: ''
+labels: 'enhancement, awaiting-triage'
+assignees: ''
+
+---
+
+## Feature Request
+
+### Problem
+A clear and concise description how this idea has manifested and the context. Elaborate on the need for this feature and/or what could be improved. Ex. I'm always frustrated when [...]
+
+### Requirements
+A clear and concise description of the requirements to satisfy the new feature. Detail what you expect from a successful implementation of the feature. Ex. When using this feature, it should [...]
+
+### Justification
+Provide the key benefits in making this a supported feature. Ex. Adding support for this feature would ensure [...]
+
+### Alternative Considerations
```

Comment:
in some cases, people might not have this, so optional?

Reply:
Based on feedback from @dimitri-yatsenko, removed all mention of optional to keep it simple. There is no way to enforce the structure anyway so the template should be treated as a guide/suggestion.